data_analysis/notebooks/businesses.ipynb

```diff
@@ -3,9 +3,20 @@
   {
    "cell_type": "code",
    "execution_count": 1,
```

Comment:
I'd probably build up a dictionary in this function as well so that the order of the items in `usefulData` does not matter. Something like

```python
     usefulData = {}
     ...
     website = soup.findAll('div', class_ = 'views-field views-field-url')[0]
     usefulData['website'] = website.find('span',class_='field-content').text
     ...
     return usefulData
```

And if you want to be fancy, you can use a NamedTuple instead of a dict. That will still let you access fields by name, but gives you the memory efficiency of a tuple.

```python
from collections import namedtuple

UsefulData = namedtuple('UsefulData', ['name', 'address', 'year_founded', ...])

...

return UsefulData(name=name_str, address=address_str, year_founded=year_founded_str, ...)
```

see https://docs.python.org/3/library/collections.html#collections.namedtuple

Reply:
Yeah, a dictionary works nicely, I wasn't sure if it was needed since it was going to be put into a CSV file. However it does help make handling headers easier, and work with later since order isn't important. I have since changed the code to be as a dictionary.